src/foam/nanos/auth/twofactor/GoogleTOTPAuthService.js

```diff
@@ -68,8 +69,10 @@ foam.CLASS({
 
         try {
           AppConfig config = (AppConfig) x.get("appConfig");
-          String path = String.format("/%s:%s", config.getName(), user.getEmail());
-          String query = String.format("secret=%s&issuer=%s&algorithm=%s", key, config.getName(), getAlgorithm());
+          SMTPEmailService service = (SMTPEmailService) x.get("smtpEmailService");
+          String name = service == null ? "nanopay Corporation" : service.getDisplayName();
```

Comment:
Oh wait, "nanopay Corporation" shouldn't be in FOAM.

Reply:
yes, good call.

Reply:
@TW80000 Replaced nanopay specific text with 'FOAM'.  Expect our nanopay side to always be configured in production.